Directory.Packages.props

```diff
@@ -1,207 +1,209 @@
-    <ItemGroup>
-      <_allowBuildFromSourcePackage Remove="@(_allowBuildFromSourcePackage)" />
-      <_sourceBuildUnexpectedPackage Remove="@(_sourceBuildUnexpectedPackage)" />
-
-      <_allowBuildFromSourcePackage Include="@(PackageReference)" Condition=" '%(PackageReference.IsImplicitlyDefined)' == 'true' " />
-
-      <_allowBuildFromSourcePackage Include="Microsoft.Build.Framework" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Build.Tasks.Core" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Build.Utilities.Core" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Build" />
-      <_allowBuildFromSourcePackage Include="Microsoft.CSharp" />
-      <_allowBuildFromSourcePackage Include="Microsoft.DotNet.Build.Tasks.Feed" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Extensions.CommandLineUtils.Sources" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Extensions.FileProviders.Abstractions" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Extensions.FileSystemGlobbing" />
-      <_allowBuildFromSourcePackage Include="Microsoft.Web.Xdt" />
-      <_allowBuildFromSourcePackage Include="Newtonsoft.Json" />
-      <_allowBuildFromSourcePackage Include="System.CommandLine" />
-      <_allowBuildFromSourcePackage Include="System.ComponentModel.Composition" />
-      <_allowBuildFromSourcePackage Include="System.Formats.Asn1" />
-      <_allowBuildFromSourcePackage Include="System.Security.Cryptography.Cng" />
-      <_allowBuildFromSourcePackage Include="System.Security.Cryptography.Pkcs" />
-      <_allowBuildFromSourcePackage Include="System.Security.Cryptography.ProtectedData" />
-      <_allowBuildFromSourcePackage Include="System.Security.Cryptography.Xml" />
-      <_allowBuildFromSourcePackage Include="System.Text.Json" />
-
-      <_sourceBuildUnexpectedPackage Include="@(PackageReference)" Exclude="@(_allowBuildFromSourcePackage)" />
-    </ItemGroup>
-    <Error Text="Found @(_sourceBuildUnexpectedPackage-&gt;Count()) PackageReferences not on DotNetBuildSourceOnly allow list: '@(_sourceBuildUnexpectedPackage)'" Condition=" '@(_sourceBuildUnexpectedPackage)' != '' " />
-  </Target>
-
-</Project>
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+
+    <MicrosoftCodeAnalysisPublicApiAnalyzersVersion Condition="'$(MicrosoftCodeAnalysisPublicApiAnalyzersVersion)' == ''">3.3.4</MicrosoftCodeAnalysisPublicApiAnalyzersVersion>
+    <NewtonsoftJsonPackageVersion Condition="'$(NewtonsoftJsonPackageVersion)' == ''">13.0.3</NewtonsoftJsonPackageVersion>
+    <SystemFormatsAsn1PackageVersion Condition="'$(SystemFormatsAsn1PackageVersion)' == ''">8.0.1</SystemFormatsAsn1PackageVersion>
+
+    <!-- Do not update the version of System.Text.Json without explicit approval from the .NET SDK and .NET MsBuild repo owners. -->

```

Comment:
At a quick glance, I was unable to identify what changes were proposed in this pull request. Hence, I am adding this review comment so that team members can understand the changes proposed. In summary, `<!-- Do not update the version of System.Text.Json without explicit approval from the .NET SDK and .NET MsBuild repo owners. -->` comment is added in this PR.

Reply:
Thanks, Kartheek. Why is this document completely modified? Was the indentation wrong before, or is it wrong now?

Please try reverting and just adding the comment to minimize the change.

Reply:
Weird... I used GitHub Web Editor, which shows the expected diff. However, GitHub itself shows that the entire file changed.

Reply:
Pressing `.` on github on this PR opens https://github.dev/NuGet/NuGet.Client/pull/6024.
From there, opening Directory.Packages.props, vscode shows at the bottom of the screen: `UTF-8 CRLF`.

Editing the URL to https://github.dev/NuGet/NuGet.Client/ and again opening Directory.Packages.props, it now says `UTF-8 LF`

So, github changed every line ending from LF to CRLF.  It did the wrong thing: https://github.com/NuGet/NuGet.Client/blob/785d8bb948a8b477e9160f6732e7df50a1bba5d1/.gitattributes#L1-L2

For what it's worth, I'm preparing a branch that will split package versions depending if the project inserts into the .NET SDK or not, because for some packages, it's necessary due to the VS SDK to avoid package downgrade warnings. I'll be adding comments similar to what I added in this release-6.11.x fix: https://github.com/NuGet/NuGet.Client/pull/6015/files

Reply:
Codeflow also shows the expected diff and shows no whitespace changes. 
![image](https://github.com/user-attachments/assets/d5a6e179-4163-4128-9278-d7ad06352fe9)

Reply:
actually, github did not do the wrong thing, the dev branch has the wrong line endings
https://github.com/NuGet/NuGet.Client/blob/785d8bb948a8b477e9160f6732e7df50a1bba5d1/.gitattributes#L62

Reply:
Unless I hear otherwise, I'm going to send a PR just to update to CRLF, then rebase this branch against that.